executor/mock/jobrunner.go

```diff
@@ -201,14 +203,21 @@ func (jobRunner *JobRunner) StopExecutorAsync() {
 }
 
 // StartJob starts a job on an existing JobRunner and returns once the job is started
-func (jobRunner *JobRunner) StartJob(jobInput *JobInput) *JobRunResponse {
+func (jobRunner *JobRunner) StartJob(jobInput *JobInput) *JobRunResponse { // nolint: gocyclo
 	// Define some stock job to run
 	var jobID string
+	attrs := make(map[string]string)
+
 	if jobInput.JobID == "" {
 		jobID = fmt.Sprintf("Titus-%v%v", r.Intn(1000), time.Now().Second())
 	} else {
 		jobID = jobInput.JobID
 	}
+
+	if jobInput.Tty {
```

Comment:
Why don't you put this after creating titus.ContainerInfo? kinda like how this gets set: https://github.com/Netflix/titus-executor/blob/master/executor/mock/jobrunner.go#L249-L251

Reply:
Good point - missed that.  Done.